### PR TITLE
Adding nosetests to mcc.py

### PR DESCRIPTION
**Pull request by Omkar20895**
This PR adds the nosetests to metadata compliance checker(mcc.py) file

### CODE REVIEW

File: podaac/nose_tests_mcc/mcc_test.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .. import mcc
+import os 
+import json 
+
+def test_check_remote_file():
+	data = mcc.check_remote_file('CF', 'https://github.com/ioos/compliance-checker/raw/master/compliance_checker/tests/data/2dim-grid.nc')
```
Comment (lewismc) [inline podaac/nose_tests_mcc/mcc_test.py]: This can me refactored into a variable

File: podaac/nose_tests_mcc/mcc_test.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+	assert data != None
+
+	data_json = json.loads(data)
+
+	assert data_json["model"] == "NETCDF4"
+
+	assert data_json["fn"] == "https://github.com/ioos/compliance-checker/raw/master/compliance_checker/tests/data/2dim-grid.nc"
```
Comment (lewismc) [inline podaac/nose_tests_mcc/mcc_test.py]: so can this

File: podaac/nose_tests_mcc/mcc_test.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+
+def test_check_local_file(): 
+	path = os.path.join(os.path.dirname(__file__), 'ascat_20130719_230600_metopa_35024_eps_o_250_2200_ovw.l2_subsetted_.nc')
```
Comment (lewismc) [inline podaac/nose_tests_mcc/mcc_test.py]: so can this

**Comment by lewismc** [comment]
brilliant @Omkar20895 thank you please see my comments

**Comment by Omkar20895** [comment]
@lewismc I have updated the PR please take a look into it. :)

File: podaac/nose_tests_mcc/mcc_test.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	url_upload = "https://github.com/ioos/compliance-checker/raw/master/compliance_checker/tests/data/2dim-grid.nc"
+	data = mcc.check_remote_file('CF', url_upload)
+	data_json = json.loads(data)
+
+	assert data != None
+
+	assert data_json["model"] == "NETCDF4"
+
+	assert data_json["fn"] == url_upload
+
```
Comment (lewismc) [inline podaac/nose_tests_mcc/mcc_test.py]: Please remove white space.

**Comment by Omkar20895** [comment]
@lewismc shall we merge this??

**Comment by lewismc** [comment]
Yes please thank you v much.

**Pull request merged**